birdhouse/config/twitcher/twitcher.ini.template

```diff
@@ -65,7 +65,7 @@ twitcher.workdir =
 twitcher.prefix =
 
 # magpie (for twitcher.adapter)
-magpie.url = https://${PAVICS_FQDN}/magpie
+magpie.url = http://magpie:2001
```

Comment:
This is not used only for performing requests. 
It is used for reporting back to the user the access point they should use or be redirected to when interacting with Magpie.
It is also used to set exposed hostname references in authorization headers/cookies. 
This change will break some logins. 

A separate config parameter could be considered to handle internal requests, but for now this is not possible.

Reply:
That's unfortunate. I think this limitation is enough to kill this PR for now until this can be resolved.